tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -443,22 +480,29 @@ function plot_variational_marginals(z, sym2range)
         if sum(length.(indices)) > 1
             offset = 1
             for r in indices
-                for j in r
-                    p = density(
-                        z[j, :]; title="$(sym)[$offset]", titlefontsize=10, label=""
-                    )
-                    push!(ps, p)
-
-                    offset += 1
-                end
+                p = density(
+                    z[r, :];
+                    title="$(sym)[$offset]",
+                    titlefontsize=10,
+                    label="",
+                    primary=false,
+                )
+                push!(ps, p)
+                offset += 1
             end
         else
-            p = density(z[first(indices), :]; title="$(sym)", titlefontsize=10, label="")
+            p = density(
+                z[first(indices), :];
+                title="$(sym)",
+                titlefontsize=10,
+                label="",
+                primary=false,
+            )
             push!(ps, p)
         end
     end
 
-    return plot(ps...; layout=(length(ps), 1), size=(500, 1500))
+    return plot(ps...; layout=(length(ps), 1), size=(500, 1500), primary=false)
```

Comment:
I tried to disable the ylabel in the original plots, as they are not properly shown. 
In [this discourse](https://discourse.julialang.org/t/suppressing-the-label-when-using-plots-or-statsplots/27658), it says this could be achieved by setting `primary=false` as an argument in the `plot` function. 
It doesn't seem to do its [job](https://julialang-buildkite-artifacts.s3.amazonaws.com/turingtutorials/4aa906f01a50c95f91b989d5027856c68f7ba9e1e218c6fc08719bf3df2a3791/html/09-variational-inference/09_variational-inference.html) though.
Any suggestions to disable the ylabel (currently the word `density` is not properly displayed) in `plot()`?

Reply:
> I tried to disable the ylabel in the original plots, as they are not properly shown.

Are you referring to the yticks being pushed out of the visible figure?

If so, those are the _ticks_, not the labels. `primary=false` has to do with the legend, not the `ylabel`. 

To fix how they're being displayed, I think you'll be better off messing around with the `margin` keyword argument, which is `1.0mm` by default. Try making this something like `2.0mm`.

> In this discourse, it says this could be achieved by setting primary=false as an argument in the plot function.

This is referring to the `label` used for the legend; not `ylabel`.

Reply:
Oh I see, so I should use `margin=2.0mm` instead. I am trying it out. Thanks for the brilliant thoughts.